### ISSUE TITLE
Compensate for lack of insert key/problem with caps lock key on Macbook

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#347]
**Reported by timeless on 2009-07-02 09:42**
This is the general keyboard layout for modern mac book pros: http://switchtoamac.com/images/hardware/macs/macbookpro/late_2008_mbp_keyboard.png

I don't know of an insert key or numlock or ...

I couldn't figure out how to use Capslock to get anything, i tried enabling capslock and then pressing caps+N and i didn't get a menu.

Windows 7 RC, NVDA 0.6p3.2
**Blocked by #225**

**Comment by leonardder** [comment]
Although I agree with bdorer that my initial closing of this ticket without motivating this was a bit too harsh, I still belief it deserves a close since #225 would provide a fix for this. I've used NVDA on a Mac Book for ages, either by just using the caps lock key or remapping the insert with sharp keys. I belief that VMware Fusion allows you to remap keys from its interface as well.

**Comment by feerrenrut** [comment]
Yes, I think its best to close this as a duplicate of #225. Most of the discussion on #225 is about how to use NVDA on a Mac with a VM anyway.

**Comment by feerrenrut** [comment]
For anyone coming across this issue, some guidance on how to setup your NVDA key on a Mac while using a VM can be found in the following article: https://www.marcozehe.de/2015/06/07/how-to-map-your-macs-capslock-key-to-a-nvda-or-jaws-key-in-a-windows-virtual-machine/
